### PR TITLE
Fix leaks in meanvar tests

### PR DESCRIPTION
**Pull request by umar456**
Fix several leaks in the meanvar tests.

### CODE REVIEW

File: test/meanvar.cpp [hunk 1]
```diff
@@ -178,20 +192,20 @@ meanvar_test_gen(string name, int in_index, int weight_index, af_var_bias bias,
   }
   if(weight_index == -1) {
       return meanvar_test<T> (name,
-                              inputs[in_index],
-                              empty,
+                              move(inputs[in_index]),
```
Comment (9prady9) [inline test/meanvar.cpp]: I don't understand why inputs has to be a vector of af_array when there is only one af_array in use in all cases. Am I missing to see something ?

Comment (9prady9) [reply test/meanvar.cpp]: Even `outputs` for that matter I think. I can see why `tests_` has to be vector of vectors but not why `outputs` has to be a vector of vectors.

File: test/meanvar.cpp [hunk 2]
```diff
@@ -137,10 +155,8 @@ meanvar_test_gen(string name, int in_index, int weight_index, af_var_bias bias,
         readTests<double, typename varOutType<double>::type, double> (TEST_DIR"/meanvar/meanvar.data", numDims_, in_, tests_);
 
         inputs.resize(in_.size());
-        for(size_t i = 0; i < in_.size(); i++) {
-          af_create_array(&inputs[i], &in_[i].front(),
-                          numDims_[i].ndims(), numDims_[i].get(), f64);
-        }
```
[the remark this answers is not in the document]
Comment (umar456) [reply test/meanvar.cpp]: Its unfortunate. The tests seem to be fast enough so speed is not an issue and if i tried to improve it; the tests would get more complicated.

File: test/meanvar.cpp (hunk 1 above)
Comment (9prady9) [reply test/meanvar.cpp]: It makes sense with the for loop back now.

**Pull request merged**